libs/compute_api/src/responses.rs

```diff
@@ -46,6 +46,39 @@ pub struct ExtensionInstallResponse {
     pub version: ExtVersion,
 }
 
+#[derive(Serialize, Clone, Copy, Debug, Default, Deserialize, PartialEq, Eq)]
+#[serde(rename_all = "snake_case")]
+pub enum PrewarmStatus {
+    #[default]
+    NotPrewarmed,
+    Prewarming,
+    Completed,
+    Failed,
+}
+
+#[derive(Serialize, Clone, Copy, Debug, Default, Deserialize, PartialEq, Eq)]
+#[serde(rename_all = "snake_case")]
+pub enum PrewarmOffloadStatus {
+    #[default]
+    NotOffloaded,
+    Offloading,
+    Completed,
+    Failed,
+}
+
+#[derive(Serialize, Default, Debug, Deserialize, Clone)]
+pub struct PrewarmState {
```

Comment:
We also discussed adding a progress into this struct, so just want to clarify that it's still the plan, and you just want to make it a separate change (which makes sense to do not bloat this PR)

Reply:
Progress already included (see PrewarmStateWithProgress)